pkgs/tools/system/netdata/ndsudo-fix-path.patch

```diff
@@ -5,13 +5,14 @@ diff --git a/src/collectors/plugins.d/ndsudo.c b/src/collectors/plugins.d/ndsudo
 index d53ca9f28..b42a121bf 100644
 --- a/src/collectors/plugins.d/ndsudo.c
 +++ b/src/collectors/plugins.d/ndsudo.c
-@@ -357,9 +357,6 @@ int main(int argc, char *argv[]) {
+@@ -357,9 +357,9 @@ int main(int argc, char *argv[]) {
          return 3;
      }
 
 -    char new_path[] = "PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin";
--    putenv(new_path);
--
++    char new_path[] = "PATH=/run/netdata/ndsudo/runtime-dependencies";
```

Comment:
Thank you for the [suggestion](https://github.com/NixOS/nixpkgs/pull/321644#discussion_r1657066925), @Izorkin!